.github/workflows/deploy.yaml

```
@@ -0,0 +1,15 @@
+name: deploy
+on:
+  push:
+    branches: [master]
```

To get deploy on specific tags, we could do this.
```suggestion
  push:
    branches: [main]
    tags: [v*]
```